### DISCUSSION TITLE
Garmin Speed Sensor 2

### DISCUSSION THREAD

**Discussion by vinfossa** [osmandapp/OsmAnd#19272] [Q&A]
Hello,

I installed the Garmin Speed Sensor 2 on my Brompton but OsmAnd doesn't give me the possibility to change the circumference of the wheel, as is done.
Waiting for your reply, Thank you

Vincenzo

**Comment by dmpr0** [comment]
Hi! **Wheel circumference** option should be available in sensor settings in Menu – Plugins – External sensors – Settings – Sensor

**Reply by vinfossa** [reply]
found thanks, but it doesn't give me the possibility to change the wheel circumference from 2,086 to 1,300

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/162695566/89276935-63a0-4f25-8b90-b3414a94a98a"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by dmpr0** [reply]
Could you please provide video of what happens when you tap on Wheel circumference?

**Reply by vinfossa** [reply]
https://github.com/osmandapp/OsmAnd/assets/162695566/05ba21b7-9151-4976-8324-6aa45b5c606d

**Comment by vinfossa** [comment]
Inviato da iPhone

**Reply by vinfossa** [reply]
https://github.com/osmandapp/OsmAnd/assets/162695566/70568484-aa10-42ff-acd9-20ecadfc7a52
